index.js

```diff
@@ -1,6 +1,9 @@
 import React from 'react';
 import { render } from 'react-dom';
 
+// TODO: This should be moved to `examples` along with `index.html` and not
+// be in real repo root (unless this is our "cloned repo" example??)
```

Comment:
This currently is a development example and will be extracted before release. Good catch!